src/main/java/org/apache/accumulo/examples/client/RowOperations.java

```diff
@@ -73,15 +76,23 @@ private static void deleteRow(String row, Connector connector, BatchWriter bw) t
     bw.flush();
   }
 
+  static class Opts extends Help {
+    @Parameter(names = "-c")
+    String clientProps = "conf/accumulo-client.properties";
+  }
+
```

Comment:
Would be nice to get back to moving examples away from the CLI stuff but I guess its OK to just get a fix in here.

Reply:
These were added for `ExamplesIT`. I am OK with CLI stuff if the options are visible and in the example and not abstracted away.  We could remove this if we want to stop testing the examples in ExamplesIT

Reply:
We could still have ExamplesIT that runs all the examples but does it programmatically instead of using the CLI.